### PR TITLE
CHANGELOG: release RC10

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

**Comment by carpawell** [comment]
Has the `Slicer` been checked?

**Comment by roman-khimov** [comment]
Yup, @smallhive has at least two projects converted now and yeah, there were #481 and #482.

**Comment by cthulhu-rider** [comment]
current `Slicer ` isn't very tasty for the node (previous generic `ObjectWriter` was more appropriate than current client-like interface), but it works and requires pretty small adoptions

**Review by cthulhu-rider — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,72 @@
...
+any API changes planned now (except deprecated APIs removal), we're just
+collecting feedback (for about a month) and improving the
+documentation. Depending on the feedback we may release RC11 in September with
+some cosmetic changes, but then a final 1.0 will be release. So please
+consider RC10 as a real stable RC and provide feedback if any.
+
+The most significant change in this new RC is unification of pool and client
+APIs. For a long time they were developed pretty much independently leading to
+differences in behavior and features, now that's fixed. Old pool APIs are
+availabe in this release to simplify transition, but will be removed before
```
Comment (carpawell) [inline CHANGELOG.md:15]:
```suggestion
available in this release to simplify transition, but will be removed before
```

Comment (roman-khimov) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,72 @@
...
+New features:
+ * additional APIs for data get/set in session package (#447)
+ * `waiter` package that implements waiting for asynchronous operations and
+   works both with Client and Pool (#451)
+ * statistics collection is available via callback both for Client and Pool
+   now, `stat` package contains an old Pool statistics data collector (#453)
+ * automatic session creation/use can be disabled for Pool now (#454)
+
+Behaviour changes:
+ * `container` package APIs using Container struct are now methods of the
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply CHANGELOG.md:28]: Client is a struct, just like Pool is. `client` is a package. "`client`'s" looks awful, `client's` is incorrect, therefore "client's"

**Review by carpawell — APPROVED** [review]

**Pull request merged**